src/Componnets/movie-card/MovieCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Poster } from "../poster/Poster";
+import "./MovieCard.css";
+
+export const MovieCard = ({
+  name,
+  poster_path,
```

Comment:
please use camelCase naming convention 
```suggestion
  posterPath,
```